package.json

```diff
@@ -58,17 +58,17 @@
     "jest": "^24.8.0",
     "jest-cli": "^24.8.0",
     "jest-localstorage-mock": "^2.4.0",
-    "lerna": "^3.14.1",
+    "lerna": "^3.15.0",
     "lint-staged": "^7.2.0",
     "npm-publish-git": "git://github.com/hedgepigdaniel/npm-publish-git.git",
     "prettier": "^1.14.2",
     "prop-types": "^15.6.2",
     "react": "^16.8.0",
     "react-dom": "^16.8.0",
     "react-hot-loader": "^4.8.8",
-    "react-redux": "^7.0.3",
+    "react-redux": "^7.1.0",
     "react-test-renderer": "^16.1.1",
-    "redux": "^3.6.0",
+    "redux": "^4.0.4",
```

Comment:
We should attempt redux 6 upgrades at some stage when we have spare time

Reply:
Where can I find out about Redux 6?